<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayWindow</class>
 <widget class="QMainWindow" name="PlayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="hit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>680</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Hit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stand">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>680</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Stand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deal">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>680</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>DEAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doubleX2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>600</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>DoubleX2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="split">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>520</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Split</string>
    </property>
   </widget>
   <widget class="QLabel" name="Score">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>550</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Score2">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>80</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bet">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="betValue">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>500</y>
      <width>250</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="endBg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>1200</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dScore">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dCard1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>80</width>
      <height>113</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dCard2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>80</width>
      <height>113</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dCard3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>80</width>
      <height>113</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dCard5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>80</width>
      <height>113</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dCard4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>80</width>
      <height>113</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>630</y>
      <width>651</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="coin5">
       <property name="styleSheet">
        <string notr="true">border-radius: 6px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/pictures/coins/5.png</normaloff>:/img/pictures/coins/5.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="coin20">
       <property name="styleSheet">
        <string notr="true">border-radius: 6px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/pictures/coins/20.png</normaloff>:/img/pictures/coins/20.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="coin50">
       <property name="styleSheet">
        <string notr="true">border-radius: 6px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/pictures/coins/50.png</normaloff>:/img/pictures/coins/50.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="coin100">
       <property name="styleSheet">
        <string notr="true">border-radius: 6px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/pictures/coins/100.png</normaloff>:/img/pictures/coins/100.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="coin500">
       <property name="styleSheet">
        <string notr="true">border-radius: 6px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/pictures/coins/500.png</normaloff>:/img/pictures/coins/500.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="link">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>271</width>
      <height>98</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Times New Roman&quot;;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="money">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>60</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1000$</string>
    </property>
   </widget>
   <widget class="QLabel" name="pCard1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>80</width>
      <height>113</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pCard2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>500</y>
      <width>80</width>
      <height>113</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pCard3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>500</y>
      <width>80</width>
      <height>113</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pCard4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>80</width>
      <height>113</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pCard5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>80</width>
      <height>113</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pScore">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>550</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="balance">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>BALANCE</string>
    </property>
   </widget>
   <widget class="QLabel" name="bet_bet">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>140</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>BET</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="link2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="oblfordeck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>80</width>
      <height>113</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>pCard1</zorder>
   <zorder>hit</zorder>
   <zorder>stand</zorder>
   <zorder>deal</zorder>
   <zorder>doubleX2</zorder>
   <zorder>split</zorder>
   <zorder>Score</zorder>
   <zorder>Score2</zorder>
   <zorder>bet</zorder>
   <zorder>betValue</zorder>
   <zorder>dScore</zorder>
   <zorder>dCard1</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>money</zorder>
   <zorder>pScore</zorder>
   <zorder>pCard2</zorder>
   <zorder>pCard3</zorder>
   <zorder>pCard4</zorder>
   <zorder>pCard5</zorder>
   <zorder>dCard2</zorder>
   <zorder>dCard3</zorder>
   <zorder>dCard4</zorder>
   <zorder>dCard5</zorder>
   <zorder>endBg</zorder>
   <zorder>link</zorder>
   <zorder>balance</zorder>
   <zorder>bet_bet</zorder>
   <zorder>name_label</zorder>
   <zorder>link2</zorder>
   <zorder>oblfordeck</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="actionquit"/>
   </widget>
   <addaction name="menuabout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionquit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
